3.29.2 breaks TypeScript builds

*Note: These issues are for bugs and feature requests for the helper libraries.
If you need help or support, please email help@twilio.com and one of our experts
will assist you!*


**Version:**
3.29.2

### Code Snippet
Just import it into a project using tcs as build command

### Exception/Log
```
api: $ tsc
api: ../../node_modules/twilio/lib/webhooks/webhooks.d.ts(1,35): error TS7016: Could not find a declaration file for module 'express'. '/code/api/node_modules/express/index.js' implicitly has an 'any' type.
api:   Try `npm install @types/express` if it exists or add a new declaration (.d.ts) file containing `declare module 'express';`
api: error Command failed with exit code 1.
```

### Steps to Reproduce
1. Create a new TSC lib
2. Import twilio 3.29.2
3. Build with tsc

--- a/package.json
+++ b/package.json
@@ -20,6 +20,7 @@
     "url": "https://github.com/twilio/twilio-node.git"
   },
   "dependencies": {
+    "@types/express": "^4.11.1",
     "deprecate": "1.0.0",
     "jsonwebtoken": "^8.1.0",
     "lodash": "^4.17.11",
@@ -31,7 +32,6 @@
     "xmlbuilder": "9.0.1"
   },
   "devDependencies": {
-    "@types/express": "^4.11.1",
     "@types/lodash": "^4.14.104",
     "@types/node": "^9.4.6",
     "eslint": "^5.14.0",

